atlasdb-impl-shared/src/main/metrics/targeted-sweep.yml

```diff
@@ -49,3 +49,8 @@ namespaces:
         tags:
           - strategy
         docs: Mean number of entries read in a batch from the targeted sweep queue.
+      sweepDelay:
+        type: gauge
+        tags:
+          - strategy
+        docs: The latest delay between iterations of targeted sweep.
```

Comment:
Can we get a numThreads metric? Does that still scale?

Reply:
We removed the thread scaling part because we can't do it without implementing some nontrivial scheduler, so it is back to using the install config

Reply:
that's a bit of a pain - presumably the change of sweep delay will mean that if someone has configured more threads than they should, they will mostly be idle (presumably desirable, as that's still less work being done)? Does this mean we should make the default more threads, or still go through a bump-when-we-need workflow?

Reply:
Yeah, they will be idle, but tbh no one really configures this unnecessarily. Bumping manually is sad, but implementing a scheduler is probably not something we want to get into right now.